documentation/CAMARA-Security-Interoperability.md

```diff
@@ -195,15 +195,18 @@ Therefore scopes should be available to API implementations.
 
 ## Missing "openid" scope
 
-[OIDC Core Authentication Request](https://openid.net/specs/openid-connect-core-1_0.html#AuthRequest) states the following about the value of scope.
+[OIDC Connect Authentication Request](https://openid.net/specs/openid-connect-core-1_0.html#AuthRequest) states the following about the value of scope.
```

Comment:
`OIDC` already includes Connect... and why do you replace "Core" with "Connect"? Another option could be:
```suggestion
[OpenID Connect Core Authentication Request](https://openid.net/specs/openid-connect-core-1_0.html#AuthRequest) states the following about the value of scope.
```
But I see it ok as it was `OIDC Core Authentication Request`